.github/workflows/ngc_reusable.yml

```diff
@@ -0,0 +1,66 @@
+name: NGC Reusable workflow
+
+on:
+  workflow_call:
+    secrets:
+      ALIYUN_DOCKER_USERNAME:
+        required: false
+      ALIYUN_DOCKER_PASSWORD:
+        required: false
+      DOCKER_USERNAME:
+        required: false
+      DOCKER_PASSWORD:
+        required: false
+
+jobs:
+  TORCH-NGC:
+    if: github.repository == 'alibaba/BladeDISC' && github.event.ref == 'refs/heads/main'
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, gpu-t4]
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Runs a single command using the runners shell
+    - name: Checkout
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        # docker pull bladedisc/bladedisc:latest-devel-pt-ngc
+        # docker build --cache-from bladedisc/bladedisc:latest-devel-pt-ngc -t disc-dev-pt-ngc \
+        docker build -t disc-dev-pt-ngc \
+          --build-arg BASEIMAGE=nvcr.io/nvidia/pytorch:21.11-py3 -f docker/dev/Dockerfile.torch.ngc .
+    - name: Build and Test DISC
+      run: |
+        set -e
+        mkdir -p $HOME/.cache/.torch/
+        mkdir -p $HOME/.cache/.torch/.cache
+        cp $HOME/.cache/remote_cache_token $HOME/.cache/.torch/.cache
+
+        nvidia-docker run --rm -t --user $(id -u) \
+          -v $HOME/.cache/.torch/:$HOME/.torch \
+          -v $HOME/.cache/.torch/.cache:$HOME/.cache \
```

Comment:
confused about above operations on `$HOME/.cache/.torch/.cache` directory.

Reply:
Bazel's local cache would be polluted if we have multiple versions of cuda, since we use cuda 11.5 in NGC which differs from the normal CI docker's image.

That's why we mount another cache directory.

Reply:
So, our cache has no knowledge about different cuda versoin? Sounds like a trouble ...

Reply:
It's only local cache will suffer the trouble.